neo/app/controllers/Application.scala

```diff
@@ -69,6 +71,7 @@ class Application @Inject()(val wsClient: WSClient,
     }
   }
 
+  // TODO delete this??
```

Comment:
NONONONO.

We should keep this one. It will be useful for people that are using HTTP. We need to provide more logical protocol for them. (In the future).